corehq/apps/reports/static/reports/v2/js/datagrid.js

```diff
@@ -40,11 +40,7 @@ hqDefine('reports/v2/js/datagrid', [
 
         self.data = options.dataModel;
         self.reportFilters = ko.observableArray(_.map(options.reportFilters, function (data) {
-            var newFilter = reportFilters.reportFilter(data);
-            newFilter.value.subscribe(function () {
-                self.data.refresh();
-            });
-            return newFilter;
+            return reportFilters.reportFilter(data);
```

Comment:
Nitpick: you can just pass `reportFilters.reportFilter` as the second param to `map` now.

Reply:
done